web/next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
```

Comment:
Can we remove this?

Reply:
If you want to use ts in next.js,you need to add this file.